Sources/ParseLiveQuery/Internal/ClientPrivate.swift

```diff
@@ -133,17 +133,17 @@ extension Client: SRWebSocketDelegate {
 
     public func webSocket(webSocket: SRWebSocket!, didFailWithError error: NSError!) {
         print("Error: \(error)")
-
-        if !disconnected {
+        //set disconnect and try reconnecting
+        if !userDisconnected{
             reconnect()
         }
     }
 
     public func webSocket(webSocket: SRWebSocket!, didCloseWithCode code: Int, reason: String!, wasClean: Bool) {
         print("code: \(code) reason: \(reason)")
-
-        // TODO: Better retry logic, unless `disconnect()` was explicitly called
```

Comment:
I feel like we should revert this comment, since this is a valid `todo` to improve the retry logic (exponential backoff and stuff).